.github/workflows/cd.yml

```diff
@@ -0,0 +1,55 @@
+name: cd
+
+on:
+  push:
+    branches: "master"
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron: "0 0 * * Tue"
+
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Setup Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.AZURE_CONTAINER_REGISTRY_URL }}
```

Comment:
I would remove this from a secret it means there's no way for anyone to tell from looking at it what it is

Reply:
Removed Secret and added Env variable containing test ACR URL. Obviously this will be updated when tested.